docs/broker-development.md

```diff
@@ -74,4 +74,23 @@ helm install akri akri-helm-charts/akri-dev \
     --set udev.configuration.discoveryDetails.udevRules[0]='KERNEL=="video[0-9]*"' \
     --set udev.configuration.brokerPod.image.repository="ghcr.io/brokers/camera-broker" \
     --set udev.configuration.brokerPod.image.tag="v0.0.1" 
+```
+
+## Specifying additional broker environment variables in a Configuration
+You can request that additional environment variables are set in Pods that request devices discovered via an Akri
+Configuration. These are set as key/value pairs in a Configuration's `brokerProperties`. For example, take the scenario
+of brokers being deployed to USB cameras discovered by Akri. You may wish to give the brokers extra image about the
```

Comment:
maybe: "... may wish to give the brokers extra **information** about the ... "

Reply:
good catch. fixed in fc2b066